lib/core/src/lib/form/services/date-format-translation.service.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Injectable } from '@angular/core';
+import { format, parse } from 'date-fns';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DateFormatTranslationService {
+    private momentToDateFnsMap = {
```

Comment:
you are using this maps in 2 places, extract to a separate class and use same import in both cases

Reply:
moved all logic to DateFnsUtils